lib/profile_api_client.rb

```diff
@@ -181,5 +195,19 @@ def delete_school_student(token:, student_id:, organisation_id:)
       response = {}
       response.deep_symbolize_keys
     end
+
+    private
+
+    def connection
+      Faraday.new(ENV.fetch('PROFILE_API_URL'))
```

Comment:
Note: We'll need to make sure the two new env vars are set in the relevant environments before deploying, because these calls to `Hash#fetch` will fail fast.